test/command-line-test.js

```diff
@@ -352,4 +352,15 @@ describe('WebPageTest Command Line', function() {
     });
   });
 
+  it('gets a test with long custom metrics script then returns API url and payload with custom metrics data', function (done) {
```

Comment:
The test fails because `data` doesn't contain the full JSON, it's trimmed at 8192 characters.
I'm not sure if it's a failure of my command line adjustments or a test suite configuration.

Reply:
Fixed the test by shortening the payload to under 6Kb.

Reply:
@lbartoli79 here is the new fixture test